src/web/mjs/connectors/BananaMecanica.mjs

```diff
@@ -0,0 +1,13 @@
+import WordPressMadaraNovel from './templates/WordPressMadaraNovel.mjs';
+
+export default class BananaMecanica extends WordPressMadaraNovel {
```

Comment:
Are there any novels on the website? If not just use `WordPressMadara`

Reply:
This one: https://leitorbm.com/manga/a-noiva-do-mal/